pep-0517.txt

```diff
@@ -162,7 +162,8 @@ key is missing, the source tree is not using this specification, and
 tools should fall back to running ``setup.py``.
 
 Where the ``build-backend`` key exists, it takes precedence over
-``setup.py``, and source trees need not include ``setup.py`` at all.
+``setup.py``, and source trees need not include ``setup.py`` at all
+if the build backend does not require it.
 Projects may still wish to include a ``setup.py`` for compatibility
 with tools that do not use this spec.
```

Comment:
I feel like the texte does not flow very well from the new line to this sentence.  Maybe merge/rephrase them?

Reply:
How about

Where the `build-backend` key exists, this takes precedence and the source tree follows the format and conventions of the specified backend (and no `setup.py` is needed unless the backend requires it).

Reply:
Amended as suggested by @pfmoore.